huxley/www/js/components/AdvisorView.js

```diff
@@ -32,21 +33,23 @@ var AdvisorView = React.createClass ({
     return (
       <div>
         <TopBar user={this.props.user} />
-        <div className="navbar rounded-top">
-          <NavTab href="/advisor/profile">
-            Profile
-          </NavTab>
-          <NavTab href="/advisor/assignments">
-            Assignments
-          </NavTab>
-          <NavTab href="/advisor/roster">
-            Delegates
-          </NavTab>
-          <NavTab href="/advisor/feedback">
-            Feedback
-          </NavTab>
-        </div>
-        {this.props.children}
+        <Shaker>
```

Comment:
If we want to use other features by animate.css, will that require us to define top level components like this as well?